common/views.py

```diff
@@ -63,7 +63,57 @@
 from opportunity.serializer import OpportunitySerializer
 from teams.models import Teams
 from teams.serializer import TeamsSerializer
+import logging
 
+# Set up logging
+logger = logging.getLogger(__name__)
+
+class UserRegistrationView(APIView):
+    """
+    User Registration View
+    """
+    
+    def post(self, request):
+        serializer = CreateUserSerializer(data=request.data)
+        if serializer.is_valid():
+            user = serializer.save()
+            # Optionally create a profile here
+            Profile.objects.create(user=user)  # Create profile for the new user
+            logger.info("User registered successfully: %s", user.email)
+            return Response({"message": "User registered successfully"}, status=status.HTTP_201_CREATED)
+        
+        logger.warning("User registration failed: %s", serializer.errors)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class UserLoginView(APIView):
```

Comment:
Mustafa has already developed this feature in the back-end. Please delete this also.